### DISCUSSION TITLE
Aws SDK Kafka :- HTTP Timeout error

### DISCUSSION THREAD

**Discussion by ChaitanyaUV** [awslabs/aws-sdk-rust#800] [Q&A] [answered]
Hi
I am trying to use AWS SDK Kafka to connect to a MSK cluster through lambda. I facing the following error :-
DispatchFailure(
DispatchFailure {
source: ConnectorError {
kind: Timeout,
source: hyper::Error(
Connect,
HttpTimeoutError {
kind: "HTTP connect",
duration: 3.1s,
},
),
connection: Unknown,
},
},
),
)

Any ideas why this might be happening ?

**Comment by jmklix** [comment, marked as answer]
Sorry for the late reply, but it's hard to tell what might be the issue. Can you provide more details about this? Specifically a minimal reproduction sample.
